examples/mqtt/device_comms/iotcc/samplePropMqtt.conf

```diff
@@ -15,6 +15,10 @@ DevicePropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "
 WebSocketUrl = "Websocket-address-url"
```

Comment:
Do we really need all these properties in samplePropMqtt.conf? We can have this file with only MQTT properties, as it is used only by MQTT examples, so that it is clear for the user.